### DISCUSSION TITLE
Can't force window to stay in light theme

### DISCUSSION THREAD

**Discussion by SamuelC0507** [tauri-apps/tauri#7108] [Q&A]
Hi guys,
my current OS theme is in dark, and when I create a window with:
```
"windows": [{
    ...
    "theme": "Light"
}]
```
The window will be in light theme at first, but after leaving it there for a few minutes, the window will turn into dark theme (the title bar will become black).
And I can actually catch the theme-change event `TauriEvent.WINDOW_THEME_CHANGED`, which I have no idea who caused it.

Are there some other settings I must set in order to force the window staying in light theme? or is there any function I can use to set the theme back to light (since I can catch the event)?

Thanks in advance!

**Comment by ahqsoftwares** [comment]
I guess the simplest answer would be to create a frameless window and make the controls with custom html

Full guide here
https://tauri.app/v1/guides/features/window-customization

**Reply by SamuelC0507** [reply]
Thanks, I guess that should work, just wanted to see if I had missed anything.
Have you encountered this issue before or by any chance know if it's a known issue? cuz I don't think I've seen other people complaining about this yet it seems quite easy to happen?

**Reply by ahqsoftwares** [reply]
Actually I haven't encountered this issue. My os is Windows 11 Beta Edition

**Reply by SamuelC0507** [reply]
Understood, I'll try to replicate this probelm on other platforms to see if it's a platform-related problem.
Thanks for your reply!

**Reply by ahqsoftwares** [reply]
It may be platform related, not sure as I am kinda lazy rn to deploy vms (I am getting too informal)

**Reply by SamuelC0507** [reply]
No I totally appreciate your input. After some tests I found out that Windows 10 seems to suffer from this prolem constantly while Windows 11 doesn't seem to be affected. Not that I can do anything about it though but was good to know the different behaviors between platforms. Thanks!
